data-products/_example/02-data-dictionary.yml

```diff
@@ -0,0 +1,79 @@
+---
+#-------------------------------------------------------------
+# Implementation notes
+#  - YAML validation required
+#  - Warn on missing recommended attributes
+#  - Warn on table & field names containing spaces or dashes
+#-------------------------------------------------------------
+databases:
```

Comment:
What's the concept of database here? It may be better to use a different term, "databases" can already refer to so many other things

Reply:
"A collection of datasets". Maybe "collections" instead of "databases"?

Reply:
Actually "collection" might imply "data collection" (as in gathering new data from users), which it absolutely isn't. I'll try "directory" to see how that lands.